EarlGrey/Action/GREYActions.h

```diff
@@ -178,6 +178,32 @@
                          xOriginStartPercentage:(CGFloat)xOriginStartPercentage
                          yOriginStartPercentage:(CGFloat)yOriginStartPercentage;
 
+/**
+ *  Returns an action that swipes through the view from a given @c startPoint to a given
+ *  @c endPoint.
+ *
+ *  @param startPoint The point where the swipe should begin.
```

Comment:
Is the startpoint based on a point in the whole screen or within the matched view?

Reply:
It is based on the whole screen currently.
But it might make more sense to base this on the origin of the matched view.
I will change that and clarify in the documentation.

Reply:
@steviki any updates on this?

Reply:
@tirodkar Yes, the change to use points based on the origin of the matched view landed in https://github.com/google/EarlGrey/pull/675/commits/0380dbb1785308b82c8b4e33065431629badf515.
So this PR is good to go from my side.

Please let me know if there are any questions left.

Reply:
QQ - isn't this the same as scroll amount? Ir are you trying to do from any point to another?